client/components/details-link/index.tsx

```diff
@@ -15,7 +15,7 @@ import { getAdminUrl } from 'wcpay/utils';
 /**
  * The parent segment is the first part of the URL after the /payments/ path.
  */
-type ParentSegment = 'deposits' | 'transactions' | 'disputes';
+type ParentSegment = 'deposits' | 'transactions';
```

Comment:
We no longer want to support '/payments/disputes/details' as a link destination.

Reply:
This component knows quite a bit about what's happening above/around it. What's its job?

Reply:
Would it be simpler to just require clients to pass the url in? Then simplify this to just rendering a link with a specific visual look & icon.

Reply:
(Not relevant for this PR, just noting so I can move on.)

Reply:
Related, `DetailsLink` exports `getDetailsURL` anyway, and it's effectively a (typed) wrapper for `getAdminUrl`. I prefer to use standard APIs throughout over wrapping (which effectively creates a second, forked API). The part I don't understand is whether `ParentSegment` is a meaningful & useful type.